HudiConnector/pom.xml

```diff
@@ -82,7 +55,7 @@ This file has been modified to include additional dependencies.
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-core</artifactId>
 			<version>${flink.version}</version>
-			<scope>compile</scope>
+			<scope>provided</scope>
 			<exclusions>
 				<exclusion>
```

Comment:
What is the impact of excluding a transitive dependency of a provided dependency? I assume no impact?